azure-iot-hub-devicesdk/samples/README.md

```diff
@@ -0,0 +1,12 @@
+# Samples for the Azure IoT Hub Device SDK
+
+This directory contains simple samples showing how to use the various features of the Microsoft Azure IoT Hub service from a device running Python
+
+## List of Samples
+
+* Send Telemetry from a Device
+* Receive Cloud-to-Device (C2D) messages
+* Send to Output from Module
```

Comment:
Receive on Input for modules

Reply:
That would be inaccurate, because that sample does not actually exist yet.